opengemini-client-common/src/main/java/io/opengemini/client/common/BaseAsyncClient.java

```diff
@@ -107,7 +107,7 @@ public CompletableFuture<Void> write(String database, Point point) {
      */
     @Override
     public CompletableFuture<Void> writeBatch(String database, List<Point> points) {
-        StringJoiner sj = new StringJoiner("\n");
+        StringJoiner sj = new StringJoiner(System.lineSeparator());
```

Comment:
It's not platform independent, it's always \n

Reply:
We will roll back this piece of code.